src/voting/weighted.ts

```diff
@@ -44,12 +44,14 @@ export default class WeightedVoting {
       ) &&
       // If voteChoice object is empty, return false
       Object.keys(voteChoice).length > 0 &&
-      // If voteChoice object values are negative number, return false
+      // If voteChoice object values have a negative number, return false
       Object.values(voteChoice).every(
         (value) => typeof value === 'number' && value >= 0
       ) &&
-      // If voteChoice is empty, return false
```

Comment:
@ChaituVR why remove this?

Reply:
@samuveth This condition already exists in [line 46](https://github.com/snapshot-labs/snapshot.js/blob/9305d5de3c9a456f792486d3d3f95328574c6036/src/voting/weighted.ts#L46), so replaced this with new one

Reply:
Approved